data/account/participationRegistry.go

```diff
@@ -169,6 +192,9 @@ type ParticipationRegistry interface {
 	// GetAll of the participation records.
 	GetAll() []ParticipationRecord
 
+	// GetWithSecrets fetches a record with all secrets for a particular round.
+	GetWithSecrets(id ParticipationID, round basics.Round) (ParticipationRecordForRound, error)
```

Comment:
Maybe GetRoundSigningKeys ?

Reply:
I'm not attached to the name, but this has SigningKeys and the StateProof key. `GetForRound` could be good if we stick with the `ParticipationForRound` struct name. `ForRound` also suggests the extra parameter which seems like a pattern in a lot of go code.